src/components/masters/program/main-content/course-cards/email-settings/data/service.js

```diff
@@ -0,0 +1,23 @@
+import qs from 'query-string';
+
+import apiClient from '../../../../../../../apiClient';
```

Comment:
😬 

I'd like to coin the phrase "the hurdles" to describe this

Reply:
Haha. Yeah, good catch. I meant to go back to try to find a way to avoid this type of import. David Joy has some prior art in another MFE that we might be able to use. I will see if I can get that working in this PR as well to avoid going up so many directories here.

Reply:
Cool. If you can't, no biggie, doesn't bother me that much. Just caught my eye haha